task.sh

```
@@ -1 +1,7 @@
 #! /bin/bash
+
+mysqldump -u "$DB_USER" -p"$DB_PASSWORD" ShopDB --no-create-db --result-file=backup.sql
```

Before invoking mysqldump, validate that DB_USER and DB_PASSWORD are set. The task requires the script to read credentials from environment variables and to exit with an informative error if they are missing. For example:

```
if [ -z "${DB_USER:-}" ] || [ -z "${DB_PASSWORD:-}" ]; then
  echo "ERROR: DB_USER and DB_PASSWORD must be set in environment" >&2
  exit 1
fi
```

Add this check near the top so the script fails early and non-interactively when variables are missing.